website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
+        var regex = /\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/;
+        var matches = content.match(/\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/g);
+        if (matches) {
+            for (var i = 0; i < matches.length; i++) {
+                var match = regex.exec(matches[i]);
+                var atwho = match[1];
+                var guid = match[3];
+                var mention = match[2];
+
+                content = content.replace(match[0], '<span class="atwho-inserted" contenteditable="false" data-atwho-guid="'+ guid + '" data-atwho-at-query="' + atwho + '">' + atwho + mention + '</span>');
+            }
+        }
+        return content.replace(/\x0D\x0A/g, '<br>');
```

Comment:
>
Can we use \r\n here instead? I think that works, and I think it reads more naturally.
>

This is what I originally tried to do and had to change it for some reason. However, from my most recent testing this can be changed to `\r\n`. It may have just been that the `\r\n` when converting from html to markdown was causing issues.